blocks/query/edit.tsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable camelcase */
+import useSWRImmutable from 'swr/immutable';
```

Comment:
@efuller Does disabling automatic revalidation eliminate the need for workarounds like this one? https://l.alley.dev/ca8db44a8f

Reply:
@dlh01 I _think it could_. `useSWRImmutable` will simply just cache an API call and unless that request changes in any way, it will continue to use the cache. 

The use case I wanted to guard against was the revalidation after switching tabs. [Here are some docs](https://swr.vercel.app/docs/revalidation#disable-automatic-revalidations) on revalidation and there are a ton of [additional options](https://swr.vercel.app/docs/api#options) as well.

I think it could be worth a try!